spec/beaker/cli_spec.rb

```diff
@@ -426,7 +426,7 @@ module Beaker
             cli.execute!
 
             copied_hosts_file = File.join(File.absolute_path(dir), 'hosts_preserved.yml')
-            expect( File.exists?(copied_hosts_file) ).to be_truthy
+            expect( File.exist?(copied_hosts_file) ).to be_truthy
```

Comment:
Line 80 and 81 above may need updating. Maybe related to the CI failure, but maybe there are more…

Maybe we can also add Ruby 3.2 to the CI matrix in `.github/workflows/test.yml`?